### PR TITLE
Node Add Update

### PR DESCRIPTION
**Pull request by darkdrgn2k**
Corrected indentation and added mandatory field of STATUS to add node json

### CODE REVIEW

File: js/map.js [hunk 1]
```diff
@@ -310,14 +310,15 @@ function customMarkerGenerateJSON() {
...
-  var sJSON = '<div class="box-header"><h2>JSON for node</h2></div><pre style="white-space: pre;margin-bottom:10px;">{\n' +
-    '  "name": "Unnamed Node",\n' +
-    '  "latitude": ' + lat + ',\n' +
-    '  "longitude":' + lng + ',\n' +
-    '  "cardinalDirection": "' + dir + '",\n' +
-    '  "floor": ' + floor + ',\n' +
-    '  "dateAdded": "' + currentJSONDate + '"\n' +
-    '}\n</pre>';
+  var sJSON = '<div class="box-header"><h2>JSON for node</h2></div><pre style="white-space: pre;margin-bottom:10px;">   {\n' +
+    '     "name": "Unnamed Node",\n' +
```
Comment (garrying) [inline js/map.js]: @darkdrgn2k there's an unused var, `name`, on line 308. Should it be used here?

File: js/map.js [hunk 2]
```diff
@@ -310,14 +310,15 @@ function customMarkerGenerateJSON() {
...
 
-  var sJSON = '<div class="box-header"><h2>JSON for node</h2></div><pre style="white-space: pre;margin-bottom:10px;">{\n' +
-    '  "name": "Unnamed Node",\n' +
-    '  "latitude": ' + lat + ',\n' +
-    '  "longitude":' + lng + ',\n' +
-    '  "cardinalDirection": "' + dir + '",\n' +
-    '  "floor": ' + floor + ',\n' +
-    '  "dateAdded": "' + currentJSONDate + '"\n' +
-    '}\n</pre>';
+  var sJSON = '<div class="box-header"><h2>JSON for node</h2></div><pre style="white-space: pre;margin-bottom:10px;">   {\n' +
```
Comment (garrying) [inline js/map.js:313]: Extra spaces after the start of `<pre>`.

Comment (darkdrgn2k) [reply js/map.js:313]: By Design

To Align with the existing nodelist format indent the first { 3 spaces

File: js/map.js (hunk 1 above)
Comment (darkdrgn2k) [reply js/map.js]: You are 100% right, just traced the variable and it is from the form. Thanks!

**Review by garrying — APPROVED** [review]

File: js/map.js (hunk 2 above)
Comment (benhylau) [reply js/map.js:313]: The 3 spaces in the node list is very strange. I think everywhere else I've seen, it's either 2 or 4 spaces indents.

Comment (darkdrgn2k) [reply js/map.js:313]: We would need to change the node list them
It was generated  by default settings using https://jsonformatter.curiousconcept.com/
And its defaulted to 3...

**Review by benhylau — APPROVED** [review]

File: js/map.js (hunk 2 above)
Comment (benhylau) [reply js/map.js:313]: If it's only me complaining let's keep it as is, maybe I just have no seen enough JSON :)

**Pull request merged**